api/base/metrics.py

```diff
@@ -0,0 +1,124 @@
+import re
+from datetime import timedelta
+
+import waffle
+from django.utils import timezone
+
+from api.base.exceptions import InvalidQueryStringError
+from osf import features
+
+
+class MetricMixin(object):
```

Comment:
A docstring would be helpful here.

Reply:
Also, maybe rename to `MetricsViewMixin` to disambiguate it from `osf.metrics.MetricMixin`?